Default (OSX).sublime-mousemap

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "button": "button1", 
+        "modifiers": ["ctrl"], 
```

Comment:
Ah, yes. I see the point. I use the mouse so infrequently in ST I forgot about the various bindings. Still, something like `["ctrl", "super"]` while a bit less convenient at least doesn't interfere with the OS or any existing bindings. I'd also note that for Linux SublimeCodeIntel [uses `"super"`](https://github.com/SublimeCodeIntel/SublimeCodeIntel/blob/master/Default%20(Linux).sublime-mousemap), though I'm not sure exactly why.